lib/Workflow/Notes/NotesAwareTrait.php

```diff
@@ -59,4 +62,22 @@ public function getNotesAdditionalFields(): array
     {
         return $this->options['notes']['additionalFields'] ?? [];
     }
+
+    /**
+     * Inject service via compiler pass.
+     * @param $customHtmlService
+     */
+    public function setCustomHtmlService($customHtmlService) {
+        if ($customHtmlService instanceof AbstractCustomHtmlService) {
```

Comment:
instanceof CustomHtmlServiceInterface

Reply:
sorry, no, actually that is fine.... But the AbstractCustomHtmlService should be a interface with it's extra functions, right?

Reply:
You mean an additional interface for ``getTransitionName()``, ``getActionName()``, ``getPosition()`` and so forth?
Don't know if this is necessary, as those fields are intended to be mainly used within the class.

Reply:
not sure honestly... I always use separate interfaces, also for this use-cases. What if a custom Symfony (not Pimcore) Workflow tries to do this?